x-pack/heartbeat/monitors/browser/project.go

```diff
@@ -94,8 +95,27 @@ func (p *Project) Close() error {
 	return nil
 }
 
-func (p *Project) extraArgs() []string {
-	extraArgs := p.projectCfg.SyntheticsArgs
+var filterMap = map[string]int{
+	"--pause-on-error":  0,
+	"--quiet-exit-code": 0,
+	"--dry-run":         0,
+	"--outfd":           1,
+	"--reporter":        1,
+	"-V":                0,
+	"--version":         0,
+	"-h":                0,
+	"--help":            0,
```

Comment:
couple of this to add 
* --inline - Would make it hard to debug when accidentally done 
* -r, --require - Modules to preload, this wont work well for project based ones
* --ws-endpoint, Listening to chrome over WS connection, wont work in service